include/NeoN/finiteVolume/cellCentred/interpolation/upwind.hpp

```diff
@@ -70,17 +79,18 @@ class Upwind : public SurfaceInterpolationFactory<ValueType>::template Register<
     }
 
     // src weight
-    void weight(const VolumeField<ValueType>&, SurfaceField<scalar>&) const override
+    void weight(const VolumeField<ValueType>& src, SurfaceField<scalar>& weights) const override
```

Comment:
remove name otherwise it will cause compiler warning.

```suggestion
    void weight(const VolumeField<ValueType>&, SurfaceField<scalar>&) const override
```